### PR TITLE
types/node: update h2 consts

### PR DESCRIPTION
**Pull request by ZaneHannanAU**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:

auto-generated by enumeration within node:

```shell
$ node
```

```js
const jkey = (key, info) => `${(typeof info).slice(0,3)} = constants.${key};`, enumkeysof = c => {const keys = Object.keys(c).sort(); return keys.reduce((s = '', k) => s + '\n' + jkey(k, c[k]), jkey(keys[0], c[keys.shift()]));}
console.log(enumkeysof(http2.constants))
```

```js
const jkey = (key, info) => `const ${key}: ${typeof info};`, enumkeysof = c => {const keys = Object.keys(c).sort(); return keys.reduce((s = '', k) => s + '\n' + jkey(k, c[k]), jkey(keys[0], c[keys.shift()]));}
console.log(enumkeysof(http2.constants))
```

### CODE REVIEW

**Comment by ZaneHannanAU** [comment]
A lot of JSX; which ... has been failing no matter what I do

File: types/node/index.d.ts [hunk 1]
```diff
@@ -8281,6 +8299,7 @@ declare module "http2" {
         const HTTP_STATUS_CONTINUE: number;
         const HTTP_STATUS_SWITCHING_PROTOCOLS: number;
         const HTTP_STATUS_PROCESSING: number;
+        const HTTP_STATUS_EARLY_HINTS: number;
```
Comment (SimonSchick) [inline types/node/index.d.ts]: Can you sort those?

**Review by SimonSchick — APPROVED** [review]

**Review by islishude — APPROVED** [review]

File: types/node/index.d.ts (hunk 1 above)
Comment (SimonSchick) [reply types/node/index.d.ts]: I think you can just sort the entire chunk.

Comment (ZaneHannanAU) [reply types/node/index.d.ts]: cool just did Object.keys(c).sort()

**Pull request closed without merging**